Automattic/wp-calypso
# Activity Log: Remove Guided Tour & Add Subheading

#### Changes proposed in this Pull Request

- Removes the Activity Log guided tour
- Adds a `subHeaderText` to the Activity Log

#### Testing instructions

In order to check that the Guided Tours have gone, you can visit these links:
Jetpack site - `/activity-log/site?tour=activityLogJetpackIntroTour`
WPcom site - `/activity-log/site?tour=activityLogWpcomIntroTour
`
This would usually trigger a Guided Tour, but verify it doesn't anymore. The sub header text should also be present. It uses the same wording as the tour for i18n benefits, but let me know if it should be changed! :) 

<img width="1049" alt="Screenshot 2020-04-02 at 08 55 13" src="https://user-images.githubusercontent.com/43215253/78224394-68433800-74c0-11ea-94de-465c4dffe50a.png">

cc @jancavan, @roccotripaldi, @sgomes

Fixes #40684
